cnxpublishing/views/__init__.py

```diff
@@ -45,6 +45,14 @@ def declare_browsable_routes(config):
     add_route('admin-moderation', '/a/moderation/')
     add_route('admin-api-keys', '/a/api-keys/')
     add_route('admin-post-publications', '/a/post-publications/')
+    add_route('admin-add-error-banner', '/a/error-banner/',
+              request_method='GET')
+    add_route('admin-add-error-banner-POST', '/a/error-banner/',
+              request_method='POST')
+    add_route('admin-edit-error-banner', '/a/error-banner/edit/{id}/',
```

Comment:
Let's drop this `/edit/` part of this url. Would you agree that it doesn't jive with any of our other urls?